HIP/hip-0000-nft-standard.md

```diff
@@ -0,0 +1,795 @@
+---
+hip: 0000
+title: Define a granular NFT Standard (Extends HIP-412)
+author: Michiel Mulders <@michielmulders>
+working-group: Michiel Mulders <@michielmulders>, Ashe Oro <@Ashe_Oro>, Hash Axis <@hashaxis>, Rocket May <@rocketmay>, HGP Patches <@HGP_Patches>
+type: Informational
+needs-council-approval: No
+status: Draft
+created: 2022-09-05
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/413#discussioncomment-3465759
+updated: 2022-09-05
+requires: 412
+---
+
+## Abstract
+
+This HIP specifies a more granular standard for non-fungible token metadata on HTS to make it easier for NFT tooling, such as NFT explorers or NFT wallets, to parse and interpret NFT metadata. 
+
+This HIP extends the active HIP 412 by using its standard scheme for non-fungible token metadata on HTS. NFTs minted using a more granular standard allow for better metadata parsing and reduce implementation nuances due to different interpretations of HIP 412.
+
+## Motivation
+
+NFT creators desire a more granular standard to define their non-fungible token metadata on HTS. The current implementation defined by HIP 412 allows for different interpretations. It's a risk for the Hedera NFT community because users will create different metadata implementations. Therefore, making it harder for NFT tooling to interpret this data correctly. 
+
+The goal is that this HIP defines a granular standard that the entire Hedera NFT community can adopt to set us up for the future.
+
+## Rationale
+
+This token metadata standard has been developed with input from multiple developers and projects in the NFT space. The goal is to provide a more granular NFT standard for "Web3 NFTs", providing a more concrete implementation for HIP-412. 
+
+This HIP focuses on:
+
+1. Providing strict guidelines for artists to define their NFT token metadata
+2. Maintaining flexibility via the `properties` object allowing for non-specified fields
+3. Maintaining a good level of interoperability and coherence with other standards such as OpenSea
+4. Providing sufficient examples for all types of NFTs to find the correct metadata implementation as an artist
+  
+## Specification
+
+Below is the human-readable schema, presented to maximize clarity. This document also includes a Formal JSON Schema definition to assist in data validation and describe the data in formal terms. **[See the Formal JSON Schema Definition](#formal-json-schema-definition) section below.**
+
+```json
+{
+	"name": "NFT name (required)",
+	"creator": "artist (required)",
+	"creatorDID": "DID URI (optional)",
+	"description": "human readable description of the asset (required)",
+	"image": "cid or path to the NFT's preview image file (required)",
+	"sha256_checksum": "SHA-256 digest of the file pointed by the image field (recommended)",
```

Comment:
Is there a reason we use sha256 over md5? md5 is faster to generate from a file and since we're not using it for storing a password and it's only used as a checksum it should do the job fine. sha256 might be overkill for this.